functions/consumer/log-consumer/src/main/java/org/springframework/cloud/fn/consumer/log/LogConsumerConfiguration.java

```diff
@@ -35,16 +35,15 @@
  *
  * @author Artem Bilan
  */
-@Configuration
+@Configuration(proxyBeanMethods = false)
 @EnableConfigurationProperties(LogConsumerProperties.class)
 public class LogConsumerConfiguration {
 
 	@Bean
 	IntegrationFlow logConsumerFlow(LogConsumerProperties logSinkProperties) {
 		return IntegrationFlows.from(MessageConsumer.class, (gateway) -> gateway.beanName("logConsumer"))
-				.handle((payload, headers) -> payload)
```

Comment:
This must be one of the "spring integration API changes" mentioned in the PR description.

Reply:
Sorry, missed this one. Will add in the next commit. The `log()` now is not a terminal operator in the flow.